test/pubsub/testdata/pubsub_sink.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: gcp-pubsub-sink
+spec:
+  vertices:
+    - name: in
+      source:
+        generator:
+          # How many messages to generate in the duration.
+          rpu: 10
+          duration: 1s
+          # Optional, size of each generated message, defaults to 10.
+          msgSize: 100
+    - name: out
+      sink:
+        udsink:
+          container:
+            image: "quay.io/numaio/numaflow-go/gcloud-pubsub-sink:latest"
+            env:
+              - name: PROJECT_ID
+                value: "pubsub-test"
+              - name: TOPIC_ID
+                value: "pubsub-test-topic"
+              - name: SUBSCRIPTION_ID
+                value: "subscription-09098ui1"
+              - name: PUBSUB_EMULATOR_HOST
+                value: "gcloud-pubsub.numaflow-system.svc.cluster.local:8681" # For production keep it blank string
```

Comment:
`# For production keep it blank string` - better move this comment to README.md and briefly explain why.